helpers/integration.go

```diff
@@ -51,6 +51,40 @@ type HACBSTestResult struct {
 	Warnings  int    `json:"warnings"`
 }
 
+HACBSTestResult := `{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"type": "object",
+	"properties": {
+	"result": {
+		"type": "string",
+		"enum": ["SUCCESS", "FAILURE", "WARNING", "SKIPPED", "ERROR"]
+		},
+	"namespace": {
+		"type": "string"
+		},
+	"timestamp": {
+		"type": "string",
+		"pattern": "^[0-9]{10}$"
+		},
+	"successes": {
+		"type": "integer",
+		"minimum": 0
+		},
+	"note": {
```

Comment:
We may also add `maxLength` to limit the length of `note` then `HACBS_TEST_OUTPUT` won't exceed 4096 bytes?

Reply:
adjusted maxLength of the note in most recent patch.